### PR TITLE
Refactor - remove live radio block mapping logic

### PR DESCRIPTION
**Pull request by jroebu14**
Changes made as part of 'making tiny changes in Simorgh' ws-media swarm refactoring. w/@richardpk @pharingee @rebeccarfmcginnbbc @jroebu14 @ryanmccombe 

**Overall change:**
- Changed live-radio rendering pattern to match the explicit import & use of components as in  `OnDemandRadio`.

**Code changes:**

- Deleted `RadioPageBlocks` logic within `LiveRadioPage`
- Added explicit importing of the components used to the `LiveRadioPage` component
- Removed `content` from `GetInitialData`, instead replacing it with the specific data required by the components. 
- Updated `GetInitialData` tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)

### CODE REVIEW

**Comment by karinathomasbbc** [comment]
Sorry I missed this swarm 😢 - looks good!

File: src/app/pages/LiveRadioPage/index.jsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import path from 'ramda/src/path';
+import { Headline } from '@bbc/psammead-headings';
+import Paragraph from '@bbc/psammead-paragraph';
 import ATIAnalytics from '../../containers/ATIAnalytics';
 import MetadataContainer from '../../containers/Metadata';
 import ChartbeatAnalytics from '../../containers/ChartbeatAnalytics';
 import Grid, { GelPageGrid } from '#app/components/Grid';
 import LinkedData from '../../containers/LinkedData';
-import RadioPageBlocks from '../../containers/RadioPageBlocks';
+import AudioPlayer from '#containers/RadioPageBlocks/Blocks/AudioPlayer';
```
Comment (karinathomasbbc) [inline src/app/pages/LiveRadioPage/index.jsx:11]: Have we considered also renaming the RadioPageBlocks container?

File: src/app/pages/LiveRadioPage/index.jsx [hunk 2]
```diff
@@ -16,9 +17,15 @@ const StyledGelPageGrid = styled(GelPageGrid)`
...
 const LiveRadioPage = ({ pageData }) => {
-  const { language, name, summary, content } = pageData;
-  const blocks = path(['blocks'], content);
-  const { dir } = useContext(ServiceContext);
+  const {
+    language,
+    name,
+    summary,
+    heading,
+    bodySummary,
```
Comment (karinathomasbbc) [inline src/app/pages/LiveRadioPage/index.jsx:25]: What's the difference between `summary` & `bodySummary`? Is it worth just renaming to `body`?

File: src/app/routes/liveRadio/getInitialData/index.test.js [hunk 3]
```diff
@@ -20,7 +20,11 @@ describe('Get initial data for live radio', () => {
     expect(pageData.summary).toEqual(
       '세계와 한반도 뉴스를 공정하고 객관적으로 전달해 드립니다',
     );
-    expect(pageData.content.blocks.length).toBeTruthy();
+    expect(pageData.heading).toEqual('BBC 코리아 라디오');
+    expect(pageData.bodySummary).toEqual(
```
Comment (karinathomasbbc) [inline src/app/routes/liveRadio/getInitialData/index.test.js:24]: Summary & body summary output look identical here - do we need both?

File: src/app/pages/LiveRadioPage/index.jsx (hunk 1 above)
Comment (rebeccamcginn) [reply src/app/pages/LiveRadioPage/index.jsx:11]: Yes, that's the focus of the next PR. We wanted to remain within the 1 hour timebox.

File: src/app/routes/liveRadio/getInitialData/index.test.js (hunk 3 above)
Comment (rebeccamcginn) [reply src/app/routes/liveRadio/getInitialData/index.test.js:24]: `promo.summary` and `content.blocks[1].text` are not guaranteed to always have the same values (although I have yet to see them vary)

Comment (karinathomasbbc) [reply src/app/routes/liveRadio/getInitialData/index.test.js:24]: OK thanks for the clarification. I think I would still like to have 1 summary and 1... _something else_ (not bodySummary as this is confusing). See https://github.com/bbc/simorgh/pull/6606#discussion_r431924530

File: src/app/pages/LiveRadioPage/index.jsx (hunk 2 above)
Comment (RichardPK) [reply src/app/pages/LiveRadioPage/index.jsx:25]: Yea there's a couple of things here - `name` == `heading`, but again - maybe something for investigation in the next swarm?

File: src/app/routes/liveRadio/getInitialData/index.test.js (hunk 3 above)
Comment (RichardPK) [reply src/app/routes/liveRadio/getInitialData/index.test.js:24]: For folk not privee to the Slack chat - we've made the decision to tackle the duplication of the same data coming from different sources in a future swarm

**Review by rebeccamcginn — APPROVED** [review]
Testing to see if I have a grey tick

File: src/app/routes/liveRadio/getInitialData/index.js [hunk 4]
```diff
@@ -16,18 +15,23 @@ const getPageIdentifier = path([
   'pageIdentifier',
 ]);
 
+const getHeading = path(['content', 'blocks', 0, 'text']);
+const getBodySummary = path(['content', 'blocks', 1, 'text']);
+const getMasterBrand = path(['content', 'blocks', 2, 'externalId']);
+
```
Comment (ryanmccombe) [inline src/app/routes/liveRadio/getInitialData/index.js]:
```suggestion

```

**Review by ryanmccombe — APPROVED** [review]

File: src/app/routes/liveRadio/getInitialData/index.js [hunk 5]
```diff
@@ -16,18 +15,21 @@ const getPageIdentifier = path([
...
 
   return {
     ...rest,
     ...(json && {
       pageData: {
-        content: addIdsToBlocks(contentData),
+        heading: getHeading(json),
+        bodySummary: getBodySummary(json),
         language: getLanguage(json),
+        masterBrand: getMasterBrand(json),
```
Comment (rebeccamcginn) [inline src/app/routes/liveRadio/getInitialData/index.js]:
```suggestion

```

**Pull request merged**